src/MongoDB/Server.c

```diff
@@ -391,11 +393,13 @@ PHP_METHOD(Server, isPassive)
 ZEND_BEGIN_ARG_INFO_EX(ai_Server_executeCommand, 0, 0, 2)
 	ZEND_ARG_INFO(0, db)
 	ZEND_ARG_OBJ_INFO(0, command, MongoDB\\Driver\\Command, 0)
+	ZEND_ARG_OBJ_INFO(0, readPreference, MongoDB\\Driver\\ReadPreference, 1)
```

Comment:
I had to double-check, but according to [this](http://lxr.php.net/xref/PHP_5_6/Zend/zend_API.h#103) the 1 is for allowing null. And the ReadPreference args here allow null, just like in Manager.